docs/core/diagnostics/snippets/OTel-Prometheus-Grafana-Yaeger/csharp/Program.cs

```diff
@@ -0,0 +1,158 @@
+﻿// <WholeProgram>
+using System.Diagnostics;
+using System.Diagnostics.Metrics;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Resources;
+using OpenTelemetry.Trace;
+using OpenTelemetry.Exporter;
+using OpenTelemetry.Logs;
+using Azure.Monitor.OpenTelemetry.AspNetCore;
+using Azure;
+
+var builder = WebApplication.CreateBuilder(args);
+
+
+// <Snippet_CustomMetrics>
+// Custom metrics for the application
+var GreeterMeter = new Meter("GOtPrGrYa.Sample", "1.0.0");
+var CountGreetings = GreeterMeter.CreateCounter<int>("greetings.count", description: "Counts the number of greetings");
+
+// Custom ActivitySource for the application
+var GreeterActivitySource = new ActivitySource("OtPrGrYa.Sample");
```

Comment:
Consider using "Example" instead of "Sample", because sample/sampler/sampling are frequently used in the telemetry/observability space for a different meaning.